stresstest/bootstrap_rtt.py

```diff
@@ -1,9 +1,15 @@
-from __future__ import print_function
-
+import time
 from os import path
 from random import randint
 from socket import gethostbyname
-import time
+
+from ipv8.community import Community, _DEFAULT_ADDRESSES, _DNS_ADDRESSES
```

Comment:
This is wrong, the `ipv8.*` imports should be after the `try .. except ..` that ensures that `ipv8.*` imports work.

Reply:
(There are more instances of this, but I'll keep it at this single comment)

Reply:
I think isort must have "fixed" this for me :disappointed: